lib/postageapp/request.rb

```diff
@@ -8,7 +8,26 @@ class PostageApp::Request
     'Accept' => 'text/json, application/json'
   }
 
-  TimeoutError = defined?(::Timeout) ? ::Timeout::Error : ::TimeoutError
+  NET_HTTP_EXCEPTIONS = [
+    Errno::ECONNABORTED,
+    Errno::ECONNREFUSED,
+    Errno::ECONNRESET,
+    Errno::EHOSTUNREACH,
+    Errno::EINVAL,
+    Errno::ENETUNREACH,
+    Errno::EPIPE,
+    IOError,
+    Net::HTTPBadResponse,
+    Net::HTTPHeaderSyntaxError,
+    Net::ProtocolError,
+    SocketError,
+    Zlib::GzipFile::Error,
+  ]
+
+  NET_HTTP_EXCEPTIONS << ::Timeout::Error if defined?(::Timeout)
```

Comment:
It looks like this no longer defers to `::TimeoutError` if `::Timeout` is not defined. Do we need to keep that behavior? Should we always include `::TimeoutError` in the `NET_HTTP_EXCEPTIONS` definition and conditionally add `::Timeout::Error`?

Reply:
Good point.  It looks like `TimeoutError` was deprecated in Ruby 2.3, and I guess `Timeout` is the replacement.  Back to the original behavior of using one or the other.